src/App.tsx

```diff
@@ -1,36 +1,20 @@
-/** @jsx jsx */
-import { Component, Fragment } from 'react'
-import { jsx } from '@emotion/core'
-import * as Hammer from 'hammerjs'
-import Info from './Info'
+import React, { Component } from 'react'
+import Hammer from 'hammerjs'
```

Comment:
Do these default imports have to be `import * as`?

Reply:
I'm not sure. What does `import * as` do? Is it a typescript thing to get a libraries types imported along with the default?

Reply:
Huh, great question -- this is running A-ok!

I was getting an error when running the old setup. I'll see if I can figure out what magic CRA is doing here